### PR TITLE
UI Changes for assessment table history AB#12436

### PR DESCRIPTION
**Pull request by pagustinqt20**
# UI Changes for assessment table history [AB#12436](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/12436)

## Description

UI Changes for assessment table history.

If you are reviewing this PR, please adhere to the guidelines as [documented](https://github.com/bcgov/healthgateway/wiki/prguidance).

## Testing

-   [ ] Unit Tests Updated
-   [ ] Functional Tests Updated
-   [x] Not Required

### UI Changes

**YES** 

image
image
image

### Browsers Tested

-   [x] Chrome
-   [ ] Safari
-   [ ] Edge
-   [ ] Firefox

Provide an explanation for any test(s) not completed.

## Notes/Additional Comments

Any additional notes or comments that may be relevant. Include any specialized deployment steps here.

## Items to Review:

-   [General PR Guidelines](https://github.com/bcgov/healthgateway/wiki/PRguidance)

-   Fulfills Work Item Requirements

    -   The changes meet/implement story's acceptance criteria. Fixes identified problems if bug.
    -   Changes not directly related to the task requirements need to be documented on the PR.

-   Compilation / Tests

    -   The changes do not break compilation and/or unit or functional tests; unless other item addresses them specifically.

-   Logic Problems / Functional Behaviour

    -   The changes work as intended and do not introduce problems.

-   Performance

    -   The changes do not introduce obvious performance issues.

-   Documented Standards

    -   The changes adhere to the team's documented [Coding Standards](https://github.com/bcgov/healthgateway/wiki/standards).

-   Readability / Maintainability
    -   The changes can be easily understood/read and allow for future enhancements without major refactoring. Readability is preferred over "clever code".
    -   Reasoning for changes needs to be clearly articulated. Disagreements should be arbitrated by a third developer.

### CODE REVIEW

**Comment by pagustinqt20** [comment]
@ninosamson why i need to use the icon? based on the mockup. there is no icon?
image

**Comment by ninosamson** [comment]
> @ninosamson why i need to use the icon? based on the mockup. there is no icon? image

You're right! Mock didn't get updated (because I didn't know how to pull it in). I'm fine without it.

**Comment by pagustinqt20** [comment]
@ninosamson If you can give me screenshot. I am okay to implement the icon.

File: Apps/AdminWebClient/src/ClientApp/src/views/CovidCard.vue [hunk 1]
```diff
@@ -660,6 +674,30 @@ export default class CovidCardView extends Vue {
                             </v-btn>
                         </v-col>
                     </v-row>
+                    <v-row dense>
+                        <v-col class="text-right">
+                            <v-btn type="submit" class="mx-2 success">
+                                <span>START COVID THERAPY ASSESSMENT</span>
+                            </v-btn>
```
Comment (MikeLyttle) [inline Apps/AdminWebClient/src/ClientApp/src/views/CovidCard.vue:681]: Nit: the text for the button should use title case to match the others.  The uppercasing is applied by the button style.

**Review by MikeLyttle — APPROVED** [review]

**Comment by pagustinqt20** [comment]
@ninosamson can I use this 
image

<i class="fas fa-clipboard-list-check"></i> - PRO License?

**Comment by ninosamson** [comment]
> @ninosamson can I use this image
> 
> fas fa-clipboard-list-check - PRO License?

Perfect, thanks @pagustinqt20

**Pull request merged**